### ISSUE TITLE
Tags area not visible at some page widths

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#3142]
After merging https://github.com/publiclab/plots2/pull/3127, I did notice one thing - in some screen widths (see screenshots), the tags are fully hidden. Is there a way to get this to reflow to the bottom at that range of widths? 

It's related to Bootstrap grid: https://getbootstrap.com/docs/3.3/css/#grid

I'll also open a new issue about this, which you're free to claim if you like! Thanks again!

![screen shot 2018-07-24 at 3 03 48 pm](https://user-images.githubusercontent.com/24359/43160410-0e493150-8f53-11e8-8dda-356a3e628fc3.png)
![screen shot 2018-07-24 at 3 03 43 pm](https://user-images.githubusercontent.com/24359/43160411-0e61e1d2-8f53-11e8-842e-621111e37f98.png)

**Comment by derek-allen** [comment]
So there's some inconsistency in how sidebars act on the various types of Nodes. Dropping the width on a notes page, the will tags disappear under a collapsible, which doesn't occur with Questions and Wikis. I'd like to try and consolidate the sidebar logic into a partial that can handle the various node cases, so that they're consistent across the site, while serving the proper functionality for each case.

**Comment by jywarren** [comment]
That'd be amazing. Thank you!!!

**Comment by derek-allen** [comment]
Is it desired for the entire sidebar to automatically collapse in sm/xs media?

**Comment by jywarren** [comment]
Well, it doesn't display nicely side by side. But perhaps it could be moved
below the content but above the comments?

**Comment by derek-allen** [comment]
The entire content of the sidebar? Or just the tagging? As it is now,
everything goes under the comments including the new post buttons (and in
the case of sidebar/related, disappears under a cheveron collapsible).
Sorry if this isn't clear, I'm trying to nail down a consistent behavior
across all sidebar logic. There's a lot to them!

I'm also seeing the sidebars on indexes and user pages that are still on
the left side and it seems a bit silly to have them bounce back and forth
navigating through the site. If it's alright, I'll move them to the right
side as well.

**Comment by jywarren** [comment]
Yes, let's standardize on the right side. Great.

I think just the tags, and there's an issue open for removing the related
content anyways... I think the most we'd want to keep is the little inset
map and "response posts" -- the rest we are hoping to achieve through the
tag links themselves, on a separate page, rather than flooding each page
with our guesses at realted content. Make sense?

**Comment by derek-allen** [comment]
Yeah that sounds good. #2493 will probably get covered by this too.
